src/FileRotator.php

```diff
@@ -156,12 +139,7 @@ public function shouldRotateFile(string $file): bool
      */
     private function rotate(string $rotateFile, string $newFile): void
     {
-        if ($this->rotateByCopy === true) {
-            copy($rotateFile, $newFile);
-        } else {
-            $this->safeRemove($newFile);
-            rename($rotateFile, $newFile);
-        }
+        copy($rotateFile, $newFile);
```

Comment:
I've meant it the other way around.

Reply:
From phpDoc around $rotateByCopy property: 
...Defaults to `true` for Windows systems that do not play well with rename...
...By setting `rotateByCopy` to `true` you can work around this problem...

I thought that I should leave the part with `true` like more general. It's not it?